skpro/regression/base/_base.py

```diff
@@ -838,3 +847,27 @@ def _check_alpha(self, alpha, name="alpha"):
             raise ValueError(f"values in {name} must be unique, but found duplicates")
 
         return alpha
+
+    def set_output(self, transform):
```

Comment:
should you not check that transform is in {"polars", "pandas", None} ?

Reply:
Good point! The reasoning I left None out as a possible transform value is because I could not think of an typical situation where the user would need to specify the transform parameter back to None. The intuition of the None being the default value was to simulate the scenario where none the `set_output` code would be used, if that makes sense. I believe that setting the default to 'pandas' would achieve the same effect to a degree

Reply:
I think it should be possible to set to the default behaviour, which is input mtype = output mtype. That could be `None`, although maybe another str might be more intuitive?

Reply:
Good idea - lets replace `None` with `defaults`. `sklearn` does the same. I'm not 100% sure what the `defaults` value in sklearn does for its `set_output` but we can follow the same convention nonetheless